src/compute/src/logging/differential.rs

```diff
@@ -290,6 +301,27 @@ impl DemuxHandler<'_, '_, '_> {
         }
     }
 
+    fn handle_batcher_event(&mut self, event: BatcherEvent) {
+        let ts = self.ts();
+        let op = event.operator;
+        let records_diff = Diff::cast_from(event.records_diff);
+        let size_diff = Diff::cast_from(event.size_diff);
+        let capacity_diff = Diff::cast_from(event.capacity_diff);
+        let allocations_diff = Diff::cast_from(event.allocations_diff);
+        self.output
+            .batcher_records
+            .give(self.cap, ((op, ()), ts, records_diff));
+        self.output
+            .batcher_size
+            .give(self.cap, ((op, ()), ts, size_diff));
+        self.output
+            .batcher_capacity
+            .give(self.cap, ((op, ()), ts, capacity_diff));
+        self.output
+            .batcher_allocations
+            .give(self.cap, ((op, ()), ts, allocations_diff));
+    }
```

Comment:
Should we also `self.notify_arrangement_size(op);` here?

Reply:
No, because differential will output any new batch on the `stream`, which we use to activate the arrangement size logging operator. The merge batcher actively reports its size, so we don't need to do anything related here.